packages/server/src/internals/procedure.ts

```diff
@@ -121,6 +114,20 @@ export class Procedure<TInputContext, TContext, TInput, TParsedInput, TOutput> {
     }
   }
 
+  private async parseOutput(rawOutput: unknown): Promise<TOutput> {
+    try {
+      if (process.env.NODE_ENV === 'production' || !this.parseOutputFn) {
+        return rawOutput as TOutput;
+      }
+      return await this.parseOutputFn(rawOutput);
```

Comment:
Would be nice if it was:

```suggestion
  return await this.parseOutputFn(rawOutput);
```

Reply:
outdated